otel-integration/k8s-helm/values.yaml

```diff
@@ -457,7 +455,6 @@ opentelemetry-cluster-collector:
   #     enabled: true
 
 kube-state-metrics:
-  enabled: true
```

Comment:
Is it supposed?

Reply:
Yes, I added line in the changelog to explain. We enable KSM based on if `extensions.kubernetesDashboard.enabled` is set in the global values (https://github.com/coralogix/telemetry-shippers/blob/master/otel-integration/k8s-helm/Chart.yaml#L25). This parameter was created initially but it's not used for anything actually.